main/main.cpp

```diff
@@ -1951,6 +1964,10 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		OS::get_singleton()->_verbose_stdout = GLOBAL_GET("debug/settings/stdout/verbose_stdout");
 	}
 
+	if (max_fps >= 0) {
```

Comment:
The CLI option is described as _A value of 0 results in unlimited framerate_, so maybe this is better:
```suggestion
	if (max_fps > 0) {
```

Reply:
I want `--max-fps 0` to be effective, in case the project setting specifies a framerate cap :slightly_smiling_face: